### ISSUE TITLE
Adversarial Tracking Control via Strongly Adaptive Online Learning with Memory. (arXiv:2102.01623v3 [cs.LG] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#16049]
Adversarial Tracking Control via Strongly Adaptive Online Learning with Memory. (arXiv:2102.01623v3 [cs.LG] UPDATED)<br>
https://ift.tt/JW1lSTV<br>
<p>We consider the problem of tracking an adversarial state sequence in a linear dynamical system subject to adversarial disturbances and loss functions, generalizing earlier settings in the literature. To this end, we develop three techniques, each of independent interest. First, we propose a comparator-adaptive algorithm for online linear optimization with movement cost. Without tuning, it nearly matches the performance of the optimally tuned gradient descent in hindsight. Next, considering a related problem called online learning with memory, we construct a novel strongly adaptive algorithm that uses our first contribution as a building block. Finally, we present the first reduction from adversarial tracking control to strongly adaptive online learning with memory. Summarizing these individual techniques, we obtain an adversarial tracking controller with a strong performance guarantee even when the reference trajectory has a large range of movement.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
